test/envtest/crds_envtest_test.go

```diff
@@ -98,14 +98,19 @@ func TestNoKongCRDsInstalledIsFatal(t *testing.T) {
 
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
-	cfg := ConfigForEnvConfig(t, envcfg)
 
 	logger := zapr.NewLogger(zap.NewNop())
-	ctrl.SetLogger(logger)
```

Comment:
Do we lose this? If yes a nasty warning with panic trace will be shown in tests logs

Reply:
Moved this to `envtest.Setup` so it's guaranteed to be called in every test case.